e2e/README.md

```diff
@@ -24,78 +21,98 @@ existsnce, sysctl settings, etc.) is as expected.
+
+## IDE Configuration
+
+### Global Settings
+
+Set `GOFLAGS="-timeout=30m -parallel=100"` in your shell configuration file.
+
+### GoLand
 
-The top-level package of the Golang E2E implementation is named `e2e_test` and is entirely separate from all AgentBaker
-packages.
+In **Run > Edit Configurations...**, set `-timeout=60m -parallel=100` in the Go tool arguments field.
 
-The `e2e_test` package has a dependency on subpackage located in the [scenario](scenario/) directory. Package `scenario`
-is where all E2E scenarios are defined, each in their own separate files. This package also defines
-common [types](scenario/types.go) related to scenario and scenario configuration, as well as the hard-coded list of SIG
-version IDs located in [images.go](scenario/images.go) used for testing different OS distros. Package `scenario` also
-contains the implementation of common cluster selectors and mutators
-within [clusterconfiguration.go](scenario/clusterconfiguration.go), though each scenario could define their own
-implementations if needed.
+### VSCode
 
-The primary testing function is located in [suite_test.go](suite_test.go), which is run by `go test ...`.
+Add to `settings.json`:
+
+```json
+{
+  "go.testFlags": ["-parallel=100"],
+  "go.testTimeout": "30m"
+}
+```
+
+## Package Structure
+
+The top-level package of the Golang E2E implementation is named `e2e_test` and is entirely separate from all AgentBaker packages.
+
+The primary testing function is located in [suite_test.go](suite_test.go), which is run by `go test`.
 
 ## E2E VHDs.
 
 Node images are pushed to Shared Image Gallery (SIG). Each image is tagged with branch name and build id.
 By default E2E tests use latest version of images from SIG with `branch=refs/heads/master` tag.
 
-### Using Arbitrary VHD Builds
+## Using VHD Images from Custom ADO Builds
 
-If you'd like to run the E2E suite using a set of VHDs built from some arbitrary run of the VHD build pipeline in the
-MSFT tenant, you can do so by specifying the SIG_VERSION_TAG_NAME and SIG_VERSION_TAG_VALUE environment variables.
-
-***NOTE: This feature can only be used with test VHD builds, using builds from official build pipeline is not supported.
-***
+Set `SIG_VERSION_TAG_NAME` and `SIG_VERSION_TAG_VALUE` to specify custom VHD builds:
 
 ```bash
 SIG_VERSION_TAG_NAME=buildId SIG_VERSION_TAG_VALUE=123456789 TAGS_TO_RUN="os=ubuntu2204" ./e2e-local.sh
 ```
 
-### Registering New VHD SKUs for E2E Testing
+### Registering New VHD SKUs
 
-When adding a new scenario which uses a VHD that doesn't currently have an associated entry in the base catalog, please
-Build and add delete-lock to the underlying image version. If it's not done, the image version will be deleted after a few days
-by the garbage collector.
+When adding tests for a new VHD image, ensure to add a delete-lock to prevent the garbage collector from deleting the image version.
```

Comment:
```suggestion
```

Reply:
Isn't it's true? I think we still need to have at least a single locked version for each image